spec/factories/course.rb

```diff
@@ -9,6 +9,7 @@
     description { 'PGCE with QTS full time' }
     course_length { 'OneYear' }
     start_date { Faker::Date.between(from: 1.month.from_now, to: 1.year.from_now) }
+    applications_open_from { CycleTimetable.apply_opens }
```

Comment:
Would it be worth randomising the `applications_open_from` date to better reflect reality?

```
{ Faker::Date.between(from: CycleTimetable.apply_opens, to: CycleTimetable.apply_opens + 2.months) }

```

Reply:
We could but then tests will flake at the beginning of the cycle unless explicitly set. Not a massive deal i guess, but we'd need to do a bit of an audit.

Reply:
Should we update the `open_on_apply` trait to set this?

Reply:
good shout have done